src/ffi/pmtrace.rs

```diff
@@ -1,136 +1,42 @@
-#![allow(unused, deref_nullptr)]
+// `pmtrace_s` struct is from generated bindgen in playermove.rs
 
-use std::mem::{align_of, size_of};
+#![allow(unused, nonstandard_style, deref_nullptr)]
+
+use std::mem::{align_of, offset_of, size_of};
 use std::os::raw::*;
-use std::ptr::null;
 
-use crate::ffi::pmplane::pmplane_t;
+use super::pmplane::pmplane_t;
+
+pub const qboolean_false_: qboolean = 0;
+pub const qboolean_true_: qboolean = 1;
+pub type qboolean = c_uint;
 
 #[repr(C)]
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug)]
 pub struct pmtrace_s {
```

Comment:
Same, if it's the same struct but with a C typedef, then get rid of one of these